<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dashboard</class>
 <widget class="QMainWindow" name="Dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>516</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dashboard</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(98, 20, 207);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="user_info">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>211</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(92, 203, 196);</string>
    </property>
    <property name="title">
     <string>thông tin người dùng</string>
    </property>
    <widget class="QLabel" name="user_name">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Tên người dùng</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>134</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="CCCD">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>CCCD :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="So_CCCD">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>123456789012</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>196</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="SDT">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>số điện thoại :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="SDT_user">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>0948598993</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>196</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>địa chỉ :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="dia_chi">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>..................</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="edit_info">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Chỉnh sửa thông tin</string>
     </property>
    </widget>
    <widget class="QWidget" name="pic_user2" native="true">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/image/Avatar-trang-den.png);</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="QL_acc">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>quản lý acc</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QL_giao_dich">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>190</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>quản lý giao dịch</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="tai_khoan_main">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>60</y>
      <width>211</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(75, 255, 180);</string>
    </property>
    <property name="title">
     <string>tài khoản</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>180</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Số Dư :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="so_du_tk">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>500000000000000</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="nap_tien">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>111</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Nạp tiền</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mua_acc">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>260</y>
       <width>111</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Mua acc</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pic_user1" native="true">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/image/Avatar-trang-den.png);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QL_tai_khoan">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Quản lý tài khoản</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>835</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
